src/components/carousel/popularCarousel/PopularCarousel.tsx

```
@@ -3,11 +3,11 @@ import { useAddWishlistV2, useRemoveWishlistV2 } from '@apis/wish';
 import PopularCard from '@components/card/popularCard/PopularCard';
 import CarouselIndex from '@components/carousel/popularCarousel/CarouselIndex';
 import ExceptLayout from '@components/except/exceptLayout/ExceptLayout';
-import useCarousel from '@hooks/useCarousel';
+import { useRouter } from 'next/navigation';
 import { useQueryClient } from '@tanstack/react-query';
```

ESLint

[error] `@tanstack/react-query` import should occur before import of `next/navigation`

(import/order)